packages/frontend/src/components/dialogs/ViewProfile/styles.module.scss

```diff
@@ -15,3 +15,17 @@
   white-space: pre-wrap;
   overflow-y: scroll;
 }
+
+.buttonWrap {
+  display: flex;
+  margin: 20px 0px;
+  justify-content: center;
+  gap: 30px;
+  button {
+    min-width: 150px;
+  }
+}
+
+.addressLine {
+  color: #666;
```

Comment:
Maybe it's just me, but I think people will be wondering why the address is darkened.

Reply:
yip, it should just have the same style as the other two lines